src/GCRealTimeMon/Utilities/CommandLineUtilities.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using CommandLine;
+using CommandLine.Text;
+using static realmon.Program;
+
+namespace realmon.Utilities
+{
+    public static class CommandLineUtilities
+    {
+        /// <summary>
+        /// Sentinel path that'll be inserted if -c has no specified value.
+        /// This value will be matched in the case of deciding which configuration to use.
+        /// This is to facilitate the behavior to overwrite the default config based on the inputs of the prompts.
+        /// </summary>
+        public const string SentinelPath = "__SENTINEL_PATH__";
+        public const string RequiredCommandNotProvided = @"
+Required Command was not provided. The monitoring begins only when a process name via -n or a process id via -p is supplied.
+Examples:
+    - GCRealTimeMon.exe -n devenv
+    - GCRealTimeMon.exe -p 1020
+";
+
+        public const string UsageDetails = @"
+Usage:
+    GCRealTimeMon.exe [command line args]
+
+More Details:
+    - Specify a process Id by using -p or a process name by using -n, the tool will show GCs as they occur in that process. If there are multiple processes with that name it would pick the first one
+    - You can specify which info to display per GC by using a config file. You can either change the current config at ``GCRealTimeMon\bin\Release\net6.0\DefaultConfig.yaml`` or specify your own by using ""-c config_file_path"", eg, ""-c c:\data\gcmon-config.yaml"" 
```

Comment:
>GCRealTimeMon\bin\Release\net6.0\DefaultConfig.yaml` [](http://example.com/codeflow?start=121&length=52)

this would need to be obtained programmatically, please see [here](https://github.com/Maoni0/realmon/blob/main/src/GCRealTimeMon/Program.cs#L183). now that I think more about it, it would need to say "if you are using this as a dotnet global tool it's the path where gcmon dll is, but if you are using this when you build it yourself, it's the DefaultConfig.yaml where GCRealTimeMon.dll is